### PR TITLE
Organize integration tests and add current status to README

### PR DESCRIPTION
**Pull request by utam0k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,25 +25,30 @@ Here is why I am rewriting a new container runtime in Rust.
...
-- [x] mount devices
-- [x] namespaces
-- [x] capabilities
-- [x] rlimits
-- [x] cgroups v1
-- [ ] cgroups v2(WIP on [#78](https://github.com/containers/youki/issues/78))
-- [ ] seccomp(WIP on [#25](https://github.com/containers/youki/issues/25))
-- [ ] hooks(WIP on [#13](https://github.com/containers/youki/issues/13))
-- [ ] rootless(WIP on [#77](https://github.com/containers/youki/issues/77))
+|   Feature    |                   Description                   |                                                State                                                |
```
Comment (Furisto) [inline README.md:39]: Youki nowadays passes much more of the oci runtime tests. You could add it to the table with the number of tests that youki currently passes or update the 'Status of youki' section.

Comment (utam0k) [reply README.md:39]: I thought too many would be hard to see, so I narrowed it down to a few. I can still add a fe2w more, so do you have any recommendations for a feature.

Comment (Furisto) [reply README.md:39]: |   Feature    |                   Description                   |                                                State                                                |
| :----------: | :---------------------------------------------: | :-------------------------------------------------------------------------------------------------: |
|    OCI Compliance    |               Compliance with [OCI Runtime Spec  ](https://github.com/opencontainers/runtime-spec)             |                                                  X out of Y test cases passing                                               |

Comment (utam0k) [reply README.md]: @Furisto PTAL

File: README.md [hunk 2]
```diff
@@ -25,25 +25,29 @@ Here is why I am rewriting a new container runtime in Rust.
...
+| :------------: | :---------------------------------------------: | :-------------------------------------------------------------------------------------------------: |
+|     Docker     |               Running via Docker                |                                                  ✅                                                  |
+|     Podman     |               Running via Podman                | It works, but cgroups isn't supported. WIP on [#24](https://github.com/containers/youki/issues/24)  |
+|   pivot_root   |            Change the root directory            |                                                  ✅                                                  |
+|     Mounts     |    Mount files and directories to container     |                                                  ✅                                                  |
+|   Namespaces   |    Mount files and directories to container     |                                                  ✅                                                  |
```
Comment (Furisto) [inline README.md:43]: This looks like a copy paste error

File: integration_test.sh [hunk 3]
```diff
@@ -2,40 +2,77 @@
...
+  # "start/start.t"
+  "state/state.t"
 )
+for case in "${test_cases[@]}"; do
+  if [[ ! -e "${ROOT}/integration_test/src/github.com/opencontainers/runtime-tools/validation/$case" ]]; then
+    GOPATH=${ROOT}/integration_test make runtimetest validation-executables
+    break
+  fi
+done
+
```
Comment (Furisto) [inline integration_test.sh:74]: make runtimetest validation-executables builds all validation executables. So why check for each of them? Does it sometimes fail?

Comment (utam0k) [reply integration_test.sh:74]: This is to allow the build to go through if all the verification executables are in place. I'd like to know if there are other ways to accomplish this.

File: README.md (hunk 2 above)
Comment (utam0k) [reply README.md]: @Furisto I fixed it.

**Review by Furisto — APPROVED** [review]

**Comment by Furisto** [comment]
Great!

**Pull request merged**